src/sources/audiosource.h

```diff
@@ -4,9 +4,8 @@
 
 #include "util/audiosignal.h"
 #include "util/indexrange.h"
-#include "util/samplebuffer.h"
 #include "util/memory.h"
-
+#include "util/samplebuffer.h"
```

Comment:
Is this a clang-format change? Scary! IMHO this is a risk to do it in a uncontrolled mass formatting.

Reply:
Include directives are grouped and sorted. This is a good thing.

Reply:
But this may break things. At least we can't do this unconditionaly without review.

Reply:
Only if there are forbidden implicit dependencies between header files. The compiler will reveal this.